### PR TITLE
Update Info Page + Partner Rewards Pages

### PR DESCRIPTION
**Pull request by valentinludu**

### CODE REVIEW

File: apps/scoutgame/app/(general)/info/partner-rewards/bountycaster/page.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import type { Metadata } from 'next';
+
+import { BountyCasterPage } from 'components/info/PartnerRewards/BountyCaster';
+
+export const metadata: Metadata = {
+  title: 'BountyCaster Partner Rewards'
+};
+
+export default async function BountyCaster() {
```
Comment (mattcasey) [inline apps/scoutgame/app/(general)/info/partner-rewards/bountycaster/page.tsx:9]: doesn't need to be async right?

File: apps/scoutgame/components/info/InfoPage.tsx [hunk 2]
```diff
@@ -10,14 +10,15 @@ export function InfoPage() {
...
+        src='/images/info/info_banner.png'
         width={854}
         height={285}
         style={{
           maxWidth: '100%',
           height: 'auto'
         }}
-        alt=''
+        priority={true}
+        alt='info banner'
```
Comment (mattcasey) [inline apps/scoutgame/components/info/InfoPage.tsx:21]: what does priority do?

File: apps/scoutgame/components/info/PartnerRewards/CeloPage.tsx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography
+} from '@mui/material';
+import Image from 'next/image';
+
+import { InfoCard } from 'components/common/DocumentPageContainer/components/InfoCard';
+import { List, ListItem } from 'components/common/DocumentPageContainer/components/List';
```
Comment (mattcasey) [inline apps/scoutgame/components/info/PartnerRewards/CeloPage.tsx:15]: nice catch to use these custom components i made, thanks

**Review by mattcasey — APPROVED** [review]

File: apps/scoutgame/components/info/InfoPage.tsx (hunk 2 above)
Comment (valentinludu) [reply apps/scoutgame/components/info/InfoPage.tsx:21]: preload the image

File: apps/scoutgame/app/(general)/info/partner-rewards/bountycaster/page.tsx (hunk 1 above)
Comment (valentinludu) [reply apps/scoutgame/app/(general)/info/partner-rewards/bountycaster/page.tsx:9]: yes

**Pull request merged**